components/provisioner/internal/provisioning/persistence/dbsession/read.go

```diff
@@ -131,6 +133,12 @@ func (r readSession) GetCluster(runtimeID string) (model.Cluster, dberrors.Error
 	}
 	cluster.ClusterConfig.OIDCConfig = &oidcConfig
 
+	dnsConfig, dberr := r.getDNSConfig(providerConfig.ID)
+	if dberr != nil {
+		return model.Cluster{}, dberr.Append("Cannot get Oidc config for runtimeID: %s", runtimeID)
```

Comment:
It's the DNS config, not the OIDC one :D

Reply:
🤦